doc/index.rst

```diff
@@ -95,6 +95,7 @@ FlamingPy is **free** and **open source**, and released under the `Apache Licens
 
    self
    quick_installation
+   install
```

Comment:
This new section and the `quick_installation` section have redundant content. Perhaps we can remove the later, even more knowing that the new section looks superb! ✨

Reply:
Actually, the `install` is essential as it is one of the top headers consistent in Xanadu software. We may want to remove `quick_installation`? @ilan-tz, what do you think?

Reply:
Done. You may want to have another look.